src/inputs/Text.vue

```diff
@@ -1,14 +1,43 @@
 <template>
-    <div class="input">
-        Input
+    <div class="text">
+        <label>{{ this.label }}</label>
+        <input v-model="superText"
+        :type="this.type" 
+        :id="this.id" 
+        :placeholder="this.placeholder"
+        :name="this.name" 
+        :value="this.value"/> 
```

Comment:
the 'value' prop is ignored when there's a 'v-model' prop present.
This library uses 'v-model' binding on the form component, and handles values for the individual component with 'value' prop and 'input' event emitting (emitting a custom event called _onChange_).
For consistency, the props should look like this:
``` javascript
<input 
    v-on:input="e => this.$emit('onChange', this.name, e.target.value)" 
    :value="this.value" />
```